model/src/main/kotlin/PackageCurationData.kt

```diff
@@ -161,4 +162,27 @@ data class PackageCurationData(
 
         return CuratedPackage(pkg, curations)
     }
+
+    /**
+     * Merge with [other] curation data. If in question, data in this instance has precedence over data in the other
+     * instance.
+     */
+    fun merge(other: PackageCurationData) =
+        PackageCurationData(
+            comment = comment ?: other.comment,
```

Comment:
The comment is supposed to "contain information about how and why the curation was created", this means important information might be lost if only one of the comments is kept. But also merging both comments might be confusing if one curation completely replaces another. So I'm not sure if merging is the right approach in general.

Reply:
Good point. I still believe joining existing comments is the only meaningful thing to do here. Esp. in the (currently only) context of creating ClearlyDefined curations, this is only something like a template after all... as part of the PR review the comment could still be reviewed / amended.

So I'll change the code to join comments.